eof/cli.py

```diff
@@ -63,6 +64,29 @@
     is_flag=True,
     help="Set logging level to DEBUG",
 )
+@click.option(
+    "--cdse-username",
+    help="Copernicus Data Space Ecosystem username. "
+    "If not provided the program asks for it",
+)
+@click.option(
+    "--cdse-password",
+    help="Copernicus Data Space Ecosystem password. "
+    "If not provided the program asks for it",
+)
+@click.option(
+    "--asf-username",
+    help="ASF username. If not provided the program asks for it",
+)
+@click.option(
+    "--asf-password",
+    help="ASF password. If not provided the program asks for it",
+)
+@click.option(
+    "--setup-netrc",
+    is_flag=True,
+    help="save credentials provided interactively in the ~/.netrc file if necessary",
+)
```

Comment:
this one is missing from the function, but `setup_netrc` is also an import. maybe we can do an absolute import, or else name this something else

Reply:
fixed